binaries/upload.py

```diff
@@ -52,40 +59,54 @@ def upload_conda_packages():
         for name in files:
             file_path = os.path.join(root, name)
             print(file_path)
-            if any(word in file_path for word in PACKAGES) and file_path.endswith("tar.bz2"):
+            if any(word in file_path for word in PACKAGES) and file_path.endswith(
+                "tar.bz2"
+            ):
                 print(f"Uploading to anaconda package: {name}")
                 anaconda_upload_command = f"anaconda -t {anaconda_token} upload --user {CONDA_USER} {file_path} --force"
-                print(f"cmd={anaconda_upload_command}")
 
                 exit_code = os.system(anaconda_upload_command)
 
                 if exit_code != 0:
                     print(f"Anaconda package upload failed for pacakge {name}")
                     return exit_code
 
-    print(f"All packages uploaded to anaconda successfully under the channel {CONDA_USER}")
+    print(
+        f"All packages uploaded to anaconda successfully under the channel {CONDA_USER}"
+    )
 
 
 if __name__ == "__main__":
     parser = argparse.ArgumentParser(
         description="Upload anaconda and pypi packages for torchserve and torch-model-archiver"
     )
     parser.add_argument(
-        "--upload-conda-packages", action="store_true", required=False, help="Specify whether to upload conda packages"
+        "--upload-conda-packages",
+        action="store_true",
+        required=False,
+        help="Specify whether to upload conda packages",
+    )
+    parser.add_argument(
+        "--upload-pypi-packages",
+        action="store_true",
+        required=False,
+        help="Specify whether to upload pypi packages",
     )
     parser.add_argument(
-        "--upload-pypi-packages", action="store_true", required=False, help="Specify whether to upload pypi packages"
+        "--test-pypi",
+        action="store_true",
+        required=False,
+        help="Specify whether to upload to test PyPI",
     )
     args = parser.parse_args()
 
     if args.upload_conda_packages:
         upload_conda_packages()
 
     if args.upload_pypi_packages:
-        upload_pypi_packages()
-    
+        upload_pypi_packages(args.test_pypi)
```

Comment:
same feedback on the two functions for this and for conda, is it possible to break those into their own functions and minimize code duplication via other intermediate functions?

Reply:
Passing args instead for some future extra args